client/components/upgrades/google-apps/google-apps-dialog/index.jsx

```diff
@@ -86,24 +90,26 @@ const GoogleAppsDialog = React.createClass( {
 				onChange={ this.setUsers }
 				onBlur={ this.save } />
 		);
-	},
+	}
 
 	setUsers( users ) {
```

Comment:
Handlers that we pass to other components/elements need be bound. React.createClass did this automatically, but here we need to do:
```jsx
setUsers = ( users ) => {
    //...
};
```